### PR TITLE
Changes are made in Dog claas, object is created in Test class

### PR DESCRIPTION
**Pull request by RosanaKmetovikj**
Changes are made and are ready for review.
Screenshot 2024-04-01 165607

### CODE REVIEW

File: src/Dog.java [hunk 1]
```diff
@@ -1,5 +1,19 @@
 public class Dog extends Animal{
-    public Dog(int age, char gender, String species) {
+    String name;
```
Comment (CevahirBb) [inline src/Dog.java:2]: I already made name variable under dog class

Comment (RosanaKmetovikj) [reply src/Dog.java:2]: okay, when we are resolvig our conflic we can keep your changes in the code.

Comment (PatelSarita) [reply src/Dog.java:2]: I have created breed variable in dog class

Comment (CevahirBb) [reply src/Dog.java:2]: yes that is okay

File: src/Test.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 public class Test {
     public static void main(String[] args) {
-
+Dog dog1=new Dog(7,'M',"Labrador","Mike");
```
Comment (PatelSarita) [inline src/Test.java:3]: I have also made dog1 object

**Review by PatelSarita — CHANGES_REQUESTED** [review]
please have a look.

File: src/Dog.java (hunk 1 above)
Comment (PatelSarita) [reply src/Dog.java:2]: ok ...thank you

**Review by CevahirBb — APPROVED** [review]

**Review by PatelSarita — COMMENTED** [review]
looks ok

**Review by PatelSarita — COMMENTED** [review]
looks fine

**Pull request merged**